website/docs/r/acm_certificate.html.markdown

```diff
@@ -0,0 +1,87 @@
+---
+layout: "aws"
+page_title: "AWS: aws_acm_certificate"
+sidebar_current: "docs-aws-resource-acm-certificate"
+description: |-
+  Requests and manages a certificate from Amazon Certificate Manager (ACM).
+---
+
+# aws_acm_certificate
+
+The ACM certificate resource allows requesting and management of certificates
+from the Amazon Certificate Manager.
+
+It deals with requesting certificates and managing their attributes and life-cycle.
+This resource does not deal with validation of a certificate but can provide inputs
+for other resources implementing the validation. It does not wait for a certificate to be issued.
+Use a [`aws_acm_certificate_validation`](acm_certificate_validation.html) resource for this.
+
+Most commonly, this resource is used to together with [`aws_route53_record`](route53_record.html) and
+[`aws_acm_certificate_validation`](acm_certificate_validation.html) to request a DNS validated certificate,
+deploy the required validation records and wait for validation to complete.
+
+## Example Usage
+
+```hcl
+resource "aws_acm_certificate" "cert" {
+  domain_name = "example.com"
+  validation_method = "DNS"
+  tags {
+    Environment = "test"
+  }
+}
+
+data "aws_route53_zone" "zone" {
```

Comment:
Removed. If I understand you correctly we leave the more complete example in the docs for validation? I think otherwise it will be hard to follow how these resources work together (e.g. that you should use the `certificate_arn` from the `aws_acm_certificate_validation` resource and not `aws_acm_certificate` if you want to make sure you depend on an issued certificate).